src/test/java/lotto/model/CounterTest.java

```diff
@@ -0,0 +1,35 @@
+package lotto.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+
+@DisplayName("step3: CounterTest")
+public class CounterTest {
+
+  @Test
+  @DisplayName("Counter 가 입력 받은 금액에 해당하는 올바른 갯수의 로또리스트를 제공하는지 확인.")
+  void counter() {
+    //given
+    String inputMoney = "14000";
+    //when
+    Counter counter = new Counter(inputMoney);
+    //then
+    assertThat(counter.getLottoList().getLottoList().size()).isEqualTo(14);
```

Comment:
```suggestion
    assertThat(counter.getLottoList().getLottoList()).hasSize(14);
```
이렇게 작성하면 더 가독성이 좋을것 같아요!

Reply:
넵 ㅎㅎ 수정하였습니다.